docs/Reference.md

```diff
@@ -17,12 +17,68 @@ Discord exposes different versions of our API. You can specify which version to
 
 ###### API Versions
 
-| Version | Status       | Default |
-| ------- | ------------ | ------- |
-| 6       | Available    | ✓       |
-| 5       | Discontinued |         |
-| 4       | Discontinued |         |
-| 3       | Discontinued |         |
+| Version | Status                           | Default |
+|---------|----------------------------------|---------|
+| 8       | Available                        |         |
+| 7       | Doesn't look like anything to me |         |
+| 6       | Deprecated                       | ✓       |
+| 5       | Discontinued                     |         |
+| 4       | Discontinued                     |         |
+| 3       | Discontinued                     |         |
+
+## Error Messages
+
+In API v8, we've improved error formatting in form error responses. The response will tell you which json key contains the error, the error code, and a human readable error message. We will be frequently adding new error messages, so a complete list of errors is not feasible and would be almost instantly out of date. Here are some examples instead:
+
+###### Array Error
+
+```json
+{
+    "code": 50035,
+    "errors": {
+        "activities": {
+            "0": {
+                "platform": {
+                    "_errors": [
```

Comment:
What does the underscore here imply? Should we not be using this? Is it subject to change?

If so, should it be documented at all? I assume it has the same implications that `_trace` does on the gateway hello?

Reply:
The underscore distinguishes the error field from the actual field names.

Reply:
Should this be documented then? In most languages, use of a leading underscore is a way of implying that something is an implementation detail, or that it should be ignored.

Reply:
Probably should be, with this concept re-visited: https://github.com/discord/discord-api-docs/pull/967/files#diff-b307d936b736f302bc7cc153f0d8a0a3R7

Reply:
@tpcstld should we expect libraries to ignore/pass through the `_error` field? We do say that `_` prefixed fields are internal only, so if that's the case, I can un-document this.

Reply:
This is a great question. @night might be better equipped as he created this error API, but I believe this is one of the cases where we have an `_` prefix but the field is public.

The reason why the `_` exists is to avoid a possible collision with a "legitimate" field named "error", due to the nested nature of the object.

Reply:
anyway that underscore note is for gateway so it doesn't apply to this

Reply:
@advaith1 is there any reason to inconsistently use notation between the REST API and the gateway going forwards, or would it be more sensible to encourage things being made consistent across the platform if possible?

I am leaning towards the latter for this, which is partially why I brought it up :)